.github/workflows/design-workflow.yml

```diff
@@ -0,0 +1,66 @@
+name: Design Changes Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      description:
+        description: 'What are we updating (eg. update button destructive color)?'
+        default: ''
+        required: true
+      jira:
+        description: 'What is the related Jira (eg. SW-1234)?'
+        default: ''
+        required: true
+      email:
+        description: 'Your terraformation email'
+        default: ''
+        required: true
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Create branch
+        run: git checkout -b $(echo design-workflow-${{ github.actor }}-${{ github.event.inputs.jira }}-$(date +%y-%m-%d-%s) | tr ' ' -)
+
+      - name: Install Dependencies
+        run: yarn install --network-timeout 100000
+
+      - name: Build dictionary 
+        run: yarn build-dictionary
+
+      - name: Bump package patch version
+        run: yarn bump patch
+
+      - name: Config user
+        run: |
+          git config --global user.email ${{ github.event.inputs.email }}
+          git config --global user.name ${{ github.actor }}
+
+      - name: Create commit message file
+        run: node .github/scripts/commit-message.js
+        env:
+          JIRA: ${{ github.event.inputs.jira }}
+          DESCRIPTION: ${{ github.event.inputs.description }}
+
+      - name: Commit changes
+        run: |
+          git add -A
+          git commit -F /tmp/commit-message
```

Comment:
Oh, good idea to do this with environment variables. In fact, I bet this eliminates the need to write out the file at all:
```
- name: Commit changes
  run: git commit -a -m "$TITLE"
  env:
    TITLE: "${{ github.event.inputs.jira }} ${{ github.event.inputs.description }}"
```

Reply:
Hmm.. good idea, let me give that a shot.

Reply:
Oops, I guess you need the separate `git add` since there might be new files, so
```
- name: Commit changes
  run: |
    git add -A
    git commit -m "$TITLE"
  env:
    TITLE: "${{ github.event.inputs.jira }} ${{ github.event.inputs.description }}"
```

Reply:
Yes, this works great. Follow up PR coming up. Thanks again for the feedback and suggestions.